__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1959,5 +1959,110 @@ describe('Image markdown conversion to html tag', () => {
         const testString = '![`code`](https://example.com/image.png)';
         const resultString = '<img src="https://example.com/image.png" alt="<code>code</code>" />';
         expect(parser.replace(testString)).toBe(resultString);
-    })
+    });
+});
+
+describe('room mentions', () => {
+    test('simple room mention', () => {
+        const testString = '#room';
+        const resultString = '<mention-report>#room</mention-report>';
+        expect(parser.replace(testString)).toBe(resultString);
+    });
+
+    test('room mention with leading word and space', () => {
+        const testString = 'hi all #room';
+        const resultString = 'hi all <mention-report>#room</mention-report>';
+        expect(parser.replace(testString)).toBe(resultString);
+    });
+
+    test('room mention with leading word and no space', () => {
+        const testString = 'hi all#room';
+        const resultString = 'hi all#room';
+        expect(parser.replace(testString)).toBe(resultString);
+    });
+
+    test('room mention with space between hash and room name', () => {
+        const testString = '# room';
+        const resultString = '<h1>room</h1>';
+        expect(parser.replace(testString)).toBe(resultString);
+    });
+
+    test.only('room mention with markdown syntax before the # prefix', () => {
```

Comment:
Did you mention to have `.only` here? I'm not entirely sure what this is used for, but it looks like we don't do that anywhere else in our tests in this repo 🤔 

```suggestion
    test('room mention with markdown syntax before the # prefix', () => {
```

Reply:
Yeah, I've commited it by mistake. Thanks for noticing it 🙇🏼

Reply:
Cool, just checking! Thanks for the quick fix!